controller/contour/namespace.go

```diff
@@ -20,20 +20,20 @@ import (
 	"context"
 	"fmt"
 
+	operatorv1alpha1 "github.com/projectcontour/contour-operator/api/v1alpha1"
+
 	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
 )
 
-const (
-	// defaultContourNamespace is the name of the namespace
-	// used by Contour.
-	defaultContourNamespace = "projectcontour"
-)
+// namespaceBlacklist is a list of namespace names that should not be removed.
+var namespaceBlacklist = []string{"contour-operator", "default"}
```

Comment:
Need a new name for this, maybe something like `coreNamespaceList` or something?

Reply:
Also should add `kube-system` to this list or have other logic to not allow specific namespaces to be created?

Reply:
@stevesloka thanks for the review. You're spot-on with both comments. I'll work on the updates and push a new commit.